tests/backup_helper.go

```diff
@@ -2028,6 +2028,8 @@ func ValidateBackup(ctx context1.Context, backupName string, orgID string, sched
 							expectedVolumeDriver = "kdmp"
 						case string(NativeCSI):
 							expectedVolumeDriver = "csi"
+						case string(NativeAzure):
```

Comment:
we can remove this and not build pipeline as native driver support is going away from 2.7.0 onwards

Reply:
I think lets keep this, from 2.7.0 onwards we just need to change the backup_type=csi in all pipelines, all other validation remains intact.

Reply:
since we are not planning to run automation/upgrade using azuredriver  or any native driver it will be simply a not used code

Reply:
Even though backup type changes, The volume driver code will still require it, let keep the native driver in case if need to test any n-1 or n-2 verisons.

Reply:
Yes you can do some basic test and merge it , we dont need a pipeline for now for native driver

Reply:
Yes we will create pipeline with csi types and test with 2.7.0 image.